### PR TITLE
Send voice messages

### PR DESCRIPTION
**Pull request by stefanceriu**
Added voice message event type, implemented and exposed method for seding voice message.

### CODE REVIEW

File: MatrixSDK/Data/MXRoom.m [hunk 1]
```diff
@@ -1433,6 +1469,8 @@ - (MXHTTPOperation*)sendFile:(NSURL*)fileLocalURL
                                                  }
                                          } mutableCopy];
     
+    [msgContent addEntriesFromDictionary:additionalTypes];
```
Comment (manuroe) [inline MatrixSDK/Data/MXRoom.m:1472]: addEntriesFromDictionary accepts only non null dictionary.

**Review by manuroe — COMMENTED** [review]
We also need to update the swift enum for message type: `MXMessageType`.

File: MatrixSDK/Data/MXRoom.m (hunk 1 above)
Comment (stefanceriu) [reply MatrixSDK/Data/MXRoom.m:1472]: 🤦

File: MatrixSDK/Data/MXRoom.m [hunk 2]
```diff
@@ -1469,7 +1469,9 @@ - (MXHTTPOperation*)_sendFile:(NSURL*)fileLocalURL
                                                  }
                                          } mutableCopy];
     
-    [msgContent addEntriesFromDictionary:additionalTypes];
+    if(additionalTypes.count) {
```
Comment (manuroe) [inline MatrixSDK/Data/MXRoom.m:1472]:
```suggestion
    if(additionalTypes.count) 
    {
```

**Review by manuroe — APPROVED** [review]

**Pull request merged**